bark/geometry/standard_shapes.cpp

```diff
@@ -31,6 +31,23 @@ Polygon bark::geometry::standard_shapes::CarRectangle() {
                            Point2d(3, -1), Point2d(-1, -1)});
 }
 
+Polygon bark::geometry::standard_shapes::GenerateCarLimousine(
+    double wheelbase, double collision_radius) {
+  double sw = collision_radius; // scale width
+  double sl = (wheelbase+2*collision_radius)/1.2906; // scale length
```

Comment:
so that we can scale the limousine shape to arbitrary wheelbases. wheen feeding the wheelbase for the original limousine shape in here, one get's back the original limousine shape again (I think there is a test for that somewhere)